doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,287 @@
+* BIN files (:file:`app_update.bin`, :file:`net_core_app_update.bin`), containing signed application firmwares for the application and network core, respectively.
+  The BIN files can be used to update running application using MCUboot bootloader, with either MCUboot serial recovery or OTA DFU using Bluetooth LE.
+
+The table below shows the relevant types of build files that are generated and the different scenarios in which they are used.
+The :file:`merged.hex` contains bootloader and firmware only for the application core, and is usually should not be used.
+
++---------------------------------+--------------------------------------+--------------------------------------------------------------------+
+| File                            | File format                          | Programming scenario                                               |
++=================================+======================================+====================================================================+
+| :file:`merged_domain.hex`       | Full image, HEX format               | Using an external debug probe and nRF Connect Programmer           |
++---------------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_update.bin`          | MCUboot compatible image, BIN format | Using the built-in bootloader and the mcumgr command line tool     |
++---------------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`net_core_app_update.bin` | MCUboot compatible image, BIN format | Using the built-in bootloader and the mcumgr command line tool     |
++---------------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`dfu_application.zip`     | MCUboot compatible image, binary     | Using nRF Programmer for Android and iOS or nRF Connect Programmer |
++---------------------------------+--------------------------------------+--------------------------------------------------------------------+
+
+There are multiple methods of programming a sample or application onto a Thingy:53.
+You can choose the method based on the availability or absence of an external debug probe to program.
+
+.. note::
+
+   If you do not have an external debug probe available to program the Thingy:53, you can directly program compatible samples to the device over USB using the built-in MCUboot bootloader and nRF Connect Programmer.
+   In this scenario, use the :file:`dfu_application.zip` file.
+
+   Some compatible samples:
+
+   * :ref:`nrf_machine_learning_app`
+   * :ref:`matter_weather_station_app`
+   * :ref:`peripheral_lbs`
+   * :ref:`peripheral_uart`
+   * :ref:`bluetooth_mesh_light`
+   * :ref:`bluetooth_mesh_light_switch`
+
+.. _thingy53_app_guide:
+
+Application guide
+=================
+
+lorem ipsum
```

Comment:
?

Reply:
@MarekPieta will add content to that section, and I'll then squash his additions.